### ISSUE TITLE
Can we fine-tune the weights directly on a new dataset instead of distilling?

### ISSUE THREAD

**Issue by WarmCongee** [NVlabs/LongLive#8]
According to this [issue](https://github.com/NVlabs/LongLive/issues/2), the recommended approach for fine-tuning or retraining from scratch is to fine-tune or train the base model and then distill it into a new autoregressive model.
If I don't need to change the prompt in a video, can I directly fine-tune the [weights](https://huggingface.co/Efficient-Large-Model/LongLive-1.3B) using the ground truth video from a new dataset? I understand this will introduce a gap between training and inference. Besides the above disadvantages, will this affect model performance in other ways?

**Comment by AndysonYs** [comment]
Great question! I haven't tried tuning on a custom dataset yet. But i think using gt video and mse loss could somehow lead to quality drop. I am very interested in this result. If you try it, we can discuss on it later.
